### PR TITLE
sysinternals@2023.03.30: Fix hash

### LINKED ISSUE #96: sysinternals@2023.03.30: hash check failed

### PR DESCRIPTION
**Pull request by BotServiceAutomation**
- Closes #96

### CODE REVIEW

**Comment by BotServiceAutomation** [comment]
All changes look good.

Wait for review from human collaborators.
### sysinternals

- [x] Description
- [x] License
- [x] Hashes
- [x] Checkver
- [x] Autoupdate

**Review by alexchapar — APPROVED** [review]

**Pull request merged**
